Nio/Matrix Wrapper/NIORoom.swift

```diff
@@ -72,12 +71,34 @@ class NIORoom: ObservableObject {
         let content = try! ReactionEvent(eventId: eventId, key: emoji).encodeContent()
         //swiftlint:disable:next redundant_optional_initialization
         var localEcho: MXEvent? = nil
-        room.sendEvent(.reaction, content: content, localEcho: &localEcho) { response in
-            print(response)
+        room.sendEvent(.reaction, content: content, localEcho: &localEcho) { _ in
             self.objectWillChange.send()
         }
     }
 
+    func edit(text: String, eventId: String) {
+        guard !text.isEmpty else { return }
+        //swiftlint:disable:next redundant_optional_initialization
+        var localEcho: MXEvent? = nil
+        let messageObject: [String: Any] = [
+            "body": "*" + text,
+            "m.new_content": [
+                "body": text,
+                "msgtype": "m.text"
```

Comment:
I'd suggest using the available constants for the message type here and below.
```suggestion
                "msgtype": kMXMessageTypeText
```

Reply:
That is much better indeed, changed it in 50d578f